src/view/pages/MyAccount/index.js

```diff
@@ -1,7 +1,12 @@
+import { useState } from "react";
 import { RiAccountCircleLine } from "react-icons/ri";
 import { Button } from "../../components";
 
 const MyAccount = () => {
+  const [toggle, setToggle] = useState(true);
+  const toggler = () => {
```

Comment:
```suggestion
  const toggleView= () => {
```

Rename it to `toggleView`